src/openshift/component.ts

```diff
@@ -565,7 +565,7 @@ export class Component extends OpenShiftItem {
             window.showInformationMessage('Import for binary OpenShift Components is not supported.');
             return;
         } else if (componentType !== ComponentType.GIT && componentType !== ComponentType.LOCAL) {
-            window.showInformationMessage(`Cannot import unknown Component type '${componentType}'.`);
+            window.showErrorMessage(`Cannot import unknown Component type '${componentType}'.`);
```

Comment:
You could use the mechanism we built into the execute function:
 - returning a string shows an info message
 - throwing an error/string shows an error message